config/cable.yml

```diff
@@ -1,8 +1,10 @@
 development:
```

Comment:
@dissident remove this file from git, add it to gitignore and create sample file instead.